### ISSUE TITLE
shpub.phar without .phar fails

### ISSUE THREAD

**Issue by cweiske** [cweiske/shpub#7]
```
$ cp dist/shpub-0.4.0.phar shpubtest
$ ./shpubtest 
PHP Warning:  require(phar:///home/cweiske/Dev/shpub/shpubtest/bin/phar-shpub.php): failed to open stream: phar error: invalid url or non-existent phar "phar:///home/cweiske/Dev/shpub/shpubtest/bin/phar-shpub.php" in /home/cweiske/Dev/shpub/shpubtest on line 24
PHP Stack trace:
PHP   1. {main}() /home/cweiske/Dev/shpub/shpubtest:0
```

**Comment by cweiske** [comment]
I actually had this problem in another of my projects and blogged about that .. and forgot that: https://cweiske.de/tagebuch/phar-renaming-no-ext.htm

Search terms I used: `php phar without extension require "failed to open"`

**Comment by cweiske** [comment]
Fixed in v0.5.0
